.gitmodules

```diff
@@ -18,3 +18,7 @@
 	path = imported/fmi-runner
 	url = https://github.com/precice/fmi-runner
 	branch = main
+[submodule "imported/su2-adapter"]
+	path = imported/su2-adapter
+	url = https://github.com/precice/su2-adapter
+	branch = docs
```

Comment:
Remember to update with:

```
git submodule set-branch --default develop imported/su2-adapter
```

when the `docs/` is in develop. Do not just edit the file.

One then needs to update the submodule, commit, and push.